README.md

```diff
@@ -70,6 +97,79 @@ Sends an event indicating that all received messages were read by the receiver.
 Parameters:
 - none
 
+```js
+AmioWebchatClient.markMessagesAsRead()
+.then(() => {
+  console.log('Messages marked as read')
+})
+.catch(err => {
+  console.log('Error while marking messages as read:', err)
+})
+```
+
+### listMessages(max, cursor)
+Loads messages from message history. Can be called multiple times to move further back in history.
+
+Parameters:
+- **max** - Number of messages to load. Should be between 1 and 100.
+- **cursor** - Reference point (message id) indicating where to start loading messages in history. If set to `null`, messages will be read from the beginning (newest first).
+
+Response format:
+- **messages** - Array of messages, sorted from newest to oldest.
+- **cursor.next** - Cursor that should be used in the next call of listMessages().
```

Comment:
Cursor pointing the very next message. Use this cursor in the next call of `listMessages()`.

Reply:
Ještě bych upravil "Curose pointing to subsequent messages. ..."